### PR TITLE
Add F17 options panel event post

### PR DESCRIPTION
**Pull request by RootLuo**
@esedward Would you mind taking a look at this pr? Thanks!

### CODE REVIEW

File: content/F2017/F17-event-options-panel.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Title: Fall 2017 Options Panel
+Date: 2017-11-08
+Category: Events
+Tags: social, panels
+Slug: options-panel-F17
+Author: Ruofan Luo
+Summary: Ask questions about options in CS
+
+Explore exciting options in CS! No idea what courses you want to take? Not sure what to do with your CS passions? Come to our Advisor Panel to find out! Talk to upper years and advisors about the options available in the CS major, as well as joint degrees and related programs.
```
Comment (annalorimer) [inline content/F2017/F17-event-options-panel.md]: Lines need to be 80chars long.

**Review by annalorimer — DISMISSED** [review]

**Comment by esedward** [comment]
@RootLuo content looks good but tbh I'm not the best person to ask to review pull requests. You can @ Anna, Evy, Fatema, or Fatima (off the top of my head but there are others) to look them over

File: content/F2017/F17-event-options-panel.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+This is a women-only event.
```
Comment (mokaixu) [inline content/F2017/F17-event-options-panel.md]: For consistency ... need the 
Who
What
When 
see http://wics.uwaterloo.ca/2017/03/dinner-with-professors-W17/ for example

dont forget to link the faq for women only

**Review by mokaixu — CHANGES_REQUESTED** [review]

**Comment by esedward** [comment]
like Monica's example, 

description I gave you

Note: This is a women-only event. (link to faq)

Event Details

* Who: Women and non-binary individuals
* Where: MC 5479
* When: Thursday September 28, 2017, 6 pm - 8 pm
* Cost: Free!

**Pull request merged**